kubectl-cloudflow/cmd/deploy_command.go

```diff
@@ -194,7 +194,7 @@ func (opts *deployOptions) deployImpl(cmd *cobra.Command, args []string) {
 	ownerReference := createOrUpdateCloudflowApplication(appClient, applicationSpec)
 
 	appInputSecret = config.UpdateSecretWithOwnerReference(ownerReference, appInputSecret)
-
+	createStreamletSecretsIfMissing(k8sClient, applicationSpec, ownerReference)
```

Comment:
adding this method in this place makes the method below to always be an update, correct?

Reply:
No that is a different secret. (input for all streamlets vs secret for streamlet). This has to be improved further TBH.